### PR TITLE
[Videoanalyzer] Fixed cspell typos in videoanalyzer

### PR DESCRIPTION
**Pull request by syso-jxx**
# Description

Fix https://github.com/Azure/azure-sdk-for-python/issues/22681

# All SDK Contribution checklist:
- [X] **The pull request does not introduce [breaking changes]**
- [X] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [X] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Deprecated. We’re retiring the Azure Video Analyzer preview service, you're ad
 
 ## 1.0.0b3 (2021-11-09)
 
-- Added device discovery and device detail request for ONVIF enabled devices.
+- Added device discovery and device detail request for ONVIF enabled devices. <!-- cspell: disable-line -->
```
Comment (kristapratico) [inline sdk/videoanalyzer/azure-media-videoanalyzer-edge/CHANGELOG.md:9]: Hi @syso-jxx, thanks so much for your PR! Would you mind moving these inline cspell ignores to the cspell.json under a filtered path? We try our best to keep cspell statements out of the source code/docs if we can.

Comment (syso-jxx) [reply sdk/videoanalyzer/azure-media-videoanalyzer-edge/CHANGELOG.md]: Thanks for your review! I moved these inline cspell ignores to the cspell.json under a filtered path.

File: sdk/videoanalyzer/azure-media-videoanalyzer-edge/samples/sample_lva.py [hunk 2]
```diff
@@ -6,8 +6,8 @@
 from azure.iot.hub.models import CloudToDeviceMethod, CloudToDeviceMethodResult
 from datetime import time
 
-device_id = os.getenv("iothub_deviceid");
-module_d = os.getenv("iothub_moduleid");
+device_id = os.getenv("iothub_deviceid"); # cspell: disable-line
+module_d = os.getenv("iothub_moduleid"); # cspell: disable-line
```
Comment (kristapratico) [inline sdk/videoanalyzer/azure-media-videoanalyzer-edge/samples/sample_lva.py:10]: Sorry I should have been more clear! Can we move all incline cspell statements in this PR to the cspell.json?

Comment (syso-jxx) [reply sdk/videoanalyzer/azure-media-videoanalyzer-edge/samples/sample_lva.py]: I moved all incline cspell statements.

**Comment by kristapratico** [comment]
CI check here: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1847868&view=results

I'll merge as soon as it's green. Thanks again! 😸

**Review by kristapratico — APPROVED** [review]

**Pull request merged**